python/train.py

```diff
@@ -18,61 +18,70 @@
 from __future__ import print_function
 
 import math
+from ffcc import model
+from ffcc import ops
+from ffcc import io
+from ffcc import input as ffcc_input
+import os
+import json
+from tensorboard.plugins.hparams import api as hp
+import tensorflow as tf
 
-from . import model
-from . import ops
-import input as ffcc_input
-import tensorflow.compat.v1 as tf
 
-# This directory stores the TFRecord for training.
-DATA_DIR = ''
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
+
+
+# This directory stores the images and metadata for training.
+DATA_DIR = '../data/shi_gehler/preprocessed/GehlerShi'
+
+# Project name
+PROJECT_NAME = 'GehlerShi'
 
 # Model dir for checkpoints, etc.
-MODEL_DIR = ''
+MODEL_DIR = './GehlerShi_model/'
 
 # The number of epochs for training.
 NUM_EPOCHS = 150
 
+# Number of testing fold in the 3-fold cross validation
+TEST_FOLD = 1 # if TEST_FOLD=0, the code will train three models, each of
+# which  will be traind with two different folds.
+
 # How often should checkpoints and summaries be saved and computed, for each
 # training epoch. Higher numbers will result in more frequent tensorboard
 # updates, at the expense of training efficiency.
 NUM_UPDATES_PER_EPOCH = 10
 
+# To print the eval results on the latest checkpoint.
+do_print_eval = False
+
 
-def train_and_eval_fn(params, hparams, train_set, eval_set):
+def train_and_eval_fn(params, hparams, train_set, train_label, eval_set,
+                      eval_label, model_dir):
   """Creates A tf.estimator and the corresponding training and eval specs.
 
   Args:
     params: dictionary of model parameters.
     hparams: dictionary of hyperparameters.
-    train_set: Name of the tfrecords to load as part of the train set.
-    eval_set: Name of the tfrecords to load as part of the eval set.
+    train_set: A list of training data dictionaries generated by
+    'io.read_dataset_from_files'.
```

Comment:
Google python style guide: indent continuation by 2 characters
" If the description is too long to fit on a single 80-character line, use a hanging indent of 2 or 4 spaces more than the parameter name "